api/envoy/service/tap/v2alpha/common.proto

```diff
@@ -24,6 +27,15 @@ message TapConfig {
   // a tap will occur and the data will be written to the configured output.
   OutputConfig output_config = 2 [(validate.rules).message.required = true];
 
+  // [#not-implemented-hide:] Specify if TAP is enabled. The % of requests\connections for which the
+  // filter is enabled.
```

Comment:
tapping is currently supported on both filters and transport sockets and this config is shared. Can you generalize the name to not include "filter"?

Reply:
what about tap_enabled?

Reply:
Sure +1